src/main/java/com/softserve/academy/spaced/repetition/service/UserCardQueueService.java

```diff
@@ -24,38 +24,44 @@ public UserCardQueueService(UserCardQueueRepository userCardQueueRepository, Use
         this.rememberingLevelRepository = rememberingLevelRepository;
     }
 
-    public UserCardQueue updateUserCardQueue(Long deckId, Long cardId, UserCardQueue userCardQueue)
+    @Transactional
+    public void updateUserCardQueue(Long deckId, Long cardId, UserCardQueueStatus userCardQueueStatus)
             throws NotAuthorisedUserException {
         User user = userService.getAuthorizedUser();
         String email = user.getAccount().getEmail();
-        UserCardQueue userCardQueueServer = userCardQueueRepository.findUserCardQueueByAccountEmailAndCardId(email, cardId);
-        RememberingLevel rememberingLevel = rememberingLevelRepository.findOne(1L);
-        if (userCardQueueServer != null) {
-            userCardQueue.setId(userCardQueueServer.getId());
-            if (userCardQueueServer.getRememberingLevel() != null) {
-                rememberingLevel = userCardQueueServer.getRememberingLevel();
-            }
+        UserCardQueue userCardQueue = userCardQueueRepository.findUserCardQueueByAccountEmailAndCardId(email, cardId);
+        if (userCardQueue != null) {
+            userCardQueue.setId(userCardQueue.getId());
```

Comment:
What is the purpose to set the id that I get?

Reply:
Thanks, it's a mistake. :)